modules/rtos/sw_services/fatfs/host/README.rst

```diff
@@ -0,0 +1,70 @@
+===================================
+FAT File System Image Creation Tool
+===================================
+
+This is the FAT file system image creation tool. This tool creates a FAT filesystem image that is populated with the contents of the specified directory. All filenames must be in 8.3 format.
+
+
+*****
```

Comment:
Applications and examples in the SDK do not need to include the SDK setup instructions.  It is safe to assume the user has the SDK setup.